modules/engines/engine-hdb/src/main/java/com/sap/xsk/hdb/ds/transformer/hdbdd/HdbddTransformer.java

```diff
@@ -121,8 +125,19 @@ public XSKDataStructureHDBTableModel transformEntitySymbolToTableModel(EntitySym
     for (int i = 0; i < entitySymbol.getElements().size(); i++) {
```

Comment:
Could you move this whole iteration into a separate private method? Something like `handlePossibleSearchIndexAnnotations()`?